velox/functions/prestosql/JsonFunctions.h

```diff
@@ -121,11 +121,12 @@ struct JsonArrayContainsFunction {
   }
 };
 
-template <typename T>
+template <typename TExec>
 struct JsonArrayLengthFunction {
-  VELOX_DEFINE_FUNCTION_TYPES(T);
+  VELOX_DEFINE_FUNCTION_TYPES(TExec);
 
-  FOLLY_ALWAYS_INLINE bool call(int64_t& len, const arg_type<Json>& json) {
+  template <typename T>
+  FOLLY_ALWAYS_INLINE bool call(T& len, const arg_type<Json>& json) {
```

Comment:
With this code change, we will need to add tests for Spark json_array_length function. Thanks.

Reply:
Done